Makefile

```diff
@@ -299,7 +299,18 @@ $(BUILDDIR)/teleport: ensure-webassets bpf-bytecode rdpclient
 TELEPORT_ARGS ?= start
 .PHONY: teleport-hot-reload
 teleport-hot-reload:
-	CompileDaemon --graceful-kill=true --exclude-dir=".git" --exclude-dir="node_modules" --build="make $(BUILDDIR)/teleport" --command="$(BUILDDIR)/teleport $(TELEPORT_ARGS)"
+	CompileDaemon \
+	  --graceful-kill=true \
+		--exclude="node_modules" \
```

Comment:
This one shouldn't be here?

Reply:
Why should it? The purpose of CompileDaemon is to monitor and reload the Go files, not the TypeScript ones. (For TypeScript, we have Vite dev server).

Reply:
We already have `--exclude-dir="node_modules" \` a bit down and here it's `exclude` (not `exclude-dir`) which is supposed to be used with files, not directories.

Reply:
Yes, I've just noticed it and deleted my comment, but you were faster :D